spec/annexes/extension_spatialindex.adoc

```diff
@@ -88,7 +88,7 @@ Each newly created spatial index SHALL be populated using the following SQL stat
 [source,sql]
 ----
 INSERT OR REPLACE INTO rtree_<t>_<c>
-  SELECT <i>, st_minx(<c>), st_maxx(<c>), st_miny(<c>), st_maxy(<c>) FROM <t>;
+  SELECT <i>, st_minx(<c>), st_maxx(<c>), st_miny(<c>), st_maxy(<c>) FROM <t> WHERE <c> NOT NULL AND NOT ST_Empty(<c>);
```

Comment:
@jyutzler @peterstace `ST_Empty` should be `ST_IsEmpty` to match the RTree Extension SQL Function specification.
The case of the existing functions should probably be corrected as well: `ST_MinX`, `ST_MaxX`, `ST_MinY`, `ST_MaxY`